setup.py

```diff
@@ -60,35 +59,6 @@
     'cherrypy.scaffold',
     'cherrypy.wsgiserver',
 ]
-data_files = [
-    ('cherrypy', [
-        'cherrypy/cherryd',
-        'cherrypy/favicon.ico',
-        'LICENSE.md',
-    ]),
-    ('cherrypy/process', []),
-    ('cherrypy/scaffold', [
-        'cherrypy/scaffold/example.conf',
-        'cherrypy/scaffold/site.conf',
-    ]),
-    ('cherrypy/scaffold/static', [
-        'cherrypy/scaffold/static/made_with_cherrypy_small.png',
-    ]),
-    ('cherrypy/test', [
-        'cherrypy/test/style.css',
-        'cherrypy/test/test.pem',
-    ]),
-    ('cherrypy/test/static', [
-        'cherrypy/test/static/index.html',
-        'cherrypy/test/static/dirback.jpg',
-    ]),
-    ('cherrypy/tutorial', [
-        'cherrypy/tutorial/tutorial.conf',
-        'cherrypy/tutorial/README.txt',
-        'cherrypy/tutorial/pdf_file.pdf',
-        'cherrypy/tutorial/custom_error.html',
-    ]),
-]
 scripts = ['cherrypy/cherryd']
```

Comment:
Nit: it would be nice to convert this into `console_scripts` entry in `entry_points`